src/api/get-devices-viewport-data.js

```diff
@@ -0,0 +1,13 @@
+import { readSync } from 'read-file-relative';
+
+
+const VIEWPORT_DATA = JSON.parse(readSync('../../data/devices.json'));
+
+ /** Gets the name and the viewport size of all devices from [this table]{@link http://viewportsizes.com/}.
+ * @function
+ * @name getDevicesViewportData
+ * @returns {ViewportData} JSON Object which contains devices name and viewport size.
```

Comment:
Please change the description according to https://github.com/DevExpress/testcafe-browser-tools/pull/147#discussion_r172150601